client/src/components/navbar.tsx

```diff
@@ -15,6 +17,10 @@ const emails = [
 ];
 
 const Navbar = () => {
+  const { status } = useSession();
+  if (status !== "authenticated") {
+    return <div></div>;
+  }
   return (
     <nav className="w-full p-6 border-b border-gray-300">
       <div className="container mx-auto flex justify-between items-center">
```

Comment:
instead of
```
         <li>
            <WorkspaceDialog />
          </li>
```

just wrap the WorkspaceDialog element itself in an `li`

e.g. if your component currently is `return <div><p /></div>`, turn that into `return <li><div><p /></div></li>`

Reply:
maybe ;)